x/twap/api_test.go

```diff
@@ -27,16 +28,18 @@ func (s *TestSuite) TestGetBeginBlockAccumulatorRecord() {
 		poolId     uint64
 		quoteDenom string
 		baseDenom  string
-		expError   bool
+		expError   error
```

Comment:
Note to reviewer: changed this to specific error type since I realized things were passing with an error, but not the right error we desired. Might want to eventually have an errors type file in twap so we can specify specific error codes instead of manually specifying expected errors here

Reply:
Yeah, we should make error types. We don't need a code though, golang `errors.Is` will suffice for our usecase here